### PR TITLE
fix: Quick fixes for Security Module

### PR DESCRIPTION
**Pull request by rodrigobersa**
#### What this PR does / why we need it:

Quick fixes for Security section.

#### Which issue(s) this PR fixes:

Fixes #
* Mostly rephrasing fixing typos and adjusting some block codes.
* Fix for Sealed Secrets section that was breaking when applying the Kustomize patch on `/workspace/modules/security/sealed-secrets/deployment.yaml`
* Adding some examples for the Pod Security Standards, under the Privileged PSS Profile.
* Changed some commands that were referring to specific resources names (e.g. Pods), for variables.

#### Quality checks

- [X] My content adheres to the style guidelines
- [ ] I ran `make test` or `make e2e-test` and it was successful

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by niallthomson — CHANGES_REQUESTED** [review]
Mostly great changes, some comments

File: website/docs/security/guardduty/findings/anonymous-user/finding2.md [hunk 1]
```diff
@@ -1,32 +1,29 @@
 ---
-title: "Discovery tacitc to access theKubernetesAPI Anonymously"
+title: "Discovery tacticskubectl delete clusterrolebinding anonymous-view to access the Kubernetes API Anonymously"
```
[the remark this answers is not in the document]
Comment (rodrigobersa) [reply website/docs/security/guardduty/findings/anonymous-user/finding2.md:2]: You're absolutely right!

File: website/docs/security/guardduty/findings/anonymous-user/finding-3.2.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ $ kubectl create rolebinding psp-access --role=psp-use --user=system:anonymous -
 
 Note that the above rolebinding command will trigger `Policy:Kubernetes/AnonymousAccessGranted` finding in guard duty within few minutes.
 
-Now let us create a pod named nginx using a HTTP post call. Make sure to replace `eks-workshop` with your **cluster name** with your cluster name and `REGION` with your region.
+Now let us create a Pod named nginx using a HTTP post call. 
 
 ```bash
-$ API_URL=`aws eks describe-cluster --name eks-workshop --query "cluster.endpoint" --region <REGION> --output text`
-$ curl -k -v  $API_URL/api/v1/namespaces/default/pods -X POST -H 'Content-Type: application/yaml'   -d '---
+$ curl -k -v $(aws eks describe-cluster --name $EKS_CLUSTER_NAME --query "cluster.endpoint" --region $AWS_DEFAULT_REGION --output text)/api/v1/namespaces/default/pods \
```
[the remark this answers is not in the document]
Comment (rodrigobersa) [reply website/docs/security/guardduty/findings/anonymous-user/finding-3.2.md:27]: Got it.

File: website/docs/security/guardduty/findings/anonymous-user/finding2.md [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
 ```bash
 $ kubectl create clusterrolebinding anonymous-view --clusterrole=view --user=system:anonymous
 ```
 
 Note that the above rolebinding command will trigger `Policy:Kubernetes/AnonymousAccessGranted` finding in guard duty within few minutes.
 
-Identify the API server url of the cluster and run a http get call for uri /api/v1/pods using curl. This is equivalent to running `kubectl get pods -A -o json`. The difference between kubectl and curl is that while using kubectl we'll be attaching an auth bearer token to authenticate however while running curl we're not using any auth bearer token and skipping authentication and using `system:anonymous` for authorization.
-
-Make sure to replace `eks-workshop` with your cluster name and `REGION` with your region.
+Identify the API server url of the cluster and run a http get call for uri /api/v1/pods using curl. This is equivalent to running `kubectl get pods -A -o json`. The difference between kubectl and curl is that while using kubectl we'll be attaching an auth bearer token to authenticate, however while running curl we're not using any auth bearer token and skipping authentication and using `system:anonymous` for authorization.
```
[the remark this answers is not in the document]
Comment (rodrigobersa) [reply website/docs/security/guardduty/findings/anonymous-user/finding2.md:16]: Absolutely.

File: website/docs/security/guardduty/findings/anonymous-user/finding2.md [hunk 4]
```diff
@@ -1,32 +1,29 @@
...
 
 ```bash
-$ API_URL=`aws eks describe-cluster --name eks-workshop --query "cluster.endpoint" --region <REGION> --output text`
-$ curl -k $API_URL/api/v1/pods
+$ curl -k $(aws eks describe-cluster --name $EKS_CLUSTER_NAME --query "cluster.endpoint" --region $AWS_DEFAULT_REGION --output text)/api/v1/pods
```
[the remark this answers is not in the document]
Comment (rodrigobersa) [reply website/docs/security/guardduty/findings/anonymous-user/finding2.md:19]: Done.

**Review by niallthomson — APPROVED** [review]

**Pull request merged**